website/docs/patterns/button-alignment/partials/specifications/grouping.md

```diff
@@ -0,0 +1,13 @@
+## Grouping
+
+How actions are grouped can influence what the user perceives as the primary action. Actions grouped correctly can alleviate decision fatigue and help users better understand the outcome of the action they take.
```

Comment:
Suggestion: 
How actions are grouped can influence what the user perceives as the primary action. Actions grouped correctly can alleviate decision fatigue and help users better understand the outcome of their actions.